crates/rustfix/src/replace.rs

```diff
@@ -312,18 +312,16 @@ mod tests {
 
     proptest! {
         #[test]
-        #[ignore]
         fn new_to_vec_roundtrip(ref s in "\\PC*") {
             assert_eq!(s.as_bytes(), Data::new(s.as_bytes()).to_vec().as_slice());
         }
 
         #[test]
-        #[ignore]
         fn replace_random_chunks(
             ref data in "\\PC*",
             ref replacements in prop::collection::vec(
                 (any::<::std::ops::Range<usize>>(), any::<Vec<u8>>()),
-                1..1337,
+                1..100,
```

Comment:
Agree that these can be un-ignored. See <https://github.com/rust-lang/rustfix/pull/67#discussion_r186228464>

Would there be an impact on test correctness for this change?
1337 took me 4s to run, and time spent seems marginal. I don't know how the nubmer is chosen. Maybe because of [this](https://en.wikipedia.org/wiki/Leet)?

Reply:
This change should not affect correctness, just the number of times we iterate on generated samples for this test.

This number is the number replacements that are generated for testing.  You got the reference to 133t, which is arbitrary.  I reduced to 100 to shorten time in test, but to still exercise it.  This reduces time for testing from 4s to <.25s.